prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -704,5 +704,30 @@ public async Task<ActionResult<string>> GetEnrolleeAdjudicationDocument(int enro
 
             return Ok(ApiResponse.Result(token));
         }
+
+        // DELETE: api/Enrollees/{enrolleeId}/adjudication-documents/{documentId}
+        /// <summary>
+        /// Delete the enrollee's adjudication document
+        /// </summary>
+        /// <param name="documentId"></param>
+        /// <returns></returns>
```

Comment:
The integration with Swagger will automatically show the request and response so we haven't been using the return annotations in the comments.